ui/src/app/views/invitation-key/component.tsx

```diff
@@ -62,8 +58,7 @@ export class InvitationKey extends React.Component<InvitationKeyProps, Invitatio
       invitationKey: this.state.invitationKey
     };
 
-    const args: FetchArgs = ProfileApiFetchParamCreator().invitationKeyVerification(request);
-    fetch(fullUrl(args.url), args.options)
+    profileApi().invitationKeyVerification(request)
```

Comment:
nit: I would just inline the request object from above - not much value in this local var (also means you can avoid importing the type).

Reply:
Updated PR as per the comment